### DISCUSSION TITLE
Sign in and Sign up with Google Documentation

### DISCUSSION THREAD

**Discussion by misskbar** [GitLiveApp/firebase-kotlin-sdk#656] [Q&A]
Any documentation about Log In/ Sign Up with Google? 
Hope you can help me!! 

Great work!

**Comment by TomislavMladenov** [comment]
I have posted an [article](https://medium.com/proandroiddev/google-sign-in-kotlin-js-f87a542e6090) about it recently, hope it helps.

**Reply by mylockerbiz** [reply]
please post the other platform examples.

**Reply by TomislavMladenov** [reply]
Here is the [wasm article]( https://medium.com/proandroiddev/kotlin-wasm-auth-part-1-google-sign-in-8287d61e82c9) for Google Sign in. 

I have published a few more with phone and apple sign in and currently I have fixed the Desktop/JVM and preparing a PR here to have it working.

**Comment by mylockerbiz** [comment]
Tomislav,
This is great. I truly appreciate this.
